source/dump.f90

```diff
@@ -1479,60 +1668,19 @@ subroutine dump_beam_population(nturn, i, ix, unit, fmt, lhighprec, loc_clo, tas
 
   ! Unrecognized format fmt
   else
-    write (lout,*) "DUMP> Format",fmt, "not understood for unit", unit
+    write (lout,"(2(a,i0))") "DUMP> ERROR Format ",fmt," not understood for unit ",unit
```

Comment:
Here you may also want to say which filename, since the unit may no longer be known by the user.

Reply:
Done.